pypeit/par/pypeitpar.py

```diff
@@ -2173,7 +2173,7 @@ def __init__(self, caldir=None, setup=None, trim=None, badpix=None, biasframe=No
         dtypes['trim'] = bool
         descr['trim'] = 'Trim the frame to isolate the data'
 
-        defaults['badpix'] = True
+        defaults['badpix'] = False
```

Comment:
If we agree that modifying `default_steps()` is preferred, we can probably remove this Parameter altogether.